### DISCUSSION TITLE
Default value for a regular config key

### DISCUSSION THREAD

**Discussion by dagap** [facebookresearch/hydra#2594] [Q&A] [answered]
WIth hydra, I can do something as follows:

```
user: ${oc.env:DB_USER, null}
```

This gives me the user specified in the environment variable or null if not found. However, I am unable to do with regular variables like:

```
quantile: ${quantile, 0.5}
```

Is something like this achievable?

**Comment by omry** [comment, marked as answer]
Check the OmegaConf select resolver documented [here](https://omegaconf.readthedocs.io/en/latest/custom_resolvers.html#oc-select).

```python
cfg = OmegaConf.create({
 "a": "Saving output to ${oc.select:output,/tmp}"
})
print(cfg.a)
# Saving output to /tmp
cfg.output = "/etc/config"
print(cfg.a)
# Saving output to /etc/config
```
